definitions/marts/looker_studio_marts/npq/ls_npqeyl_enrolments.sqlx

```diff
@@ -1,15 +1,15 @@
 config {
     type: "table",
     assertions: {
-        uniqueKey: ["application_ecf_id"]

```

Comment:
Same question.

Reply:
It is the same as the V2 file you approved previously.